### ISSUE TITLE
Removing Target

### ISSUE THREAD

**Issue by dustinhudson** [dwonderley/mookAI#18]
The current behavior of the module is to remove the target after approving the action. This is causing compatibility issues with Midi-QOL for automating damage. Having an option to clear or not clear the target would solve the issue I believe. I tested a quick proof by commenting out the clearTargets() function in the developer window and Midi-QOL was able to correctly assign the damage as normal.

**Comment by shinkei13** [comment]
I'd like to second this if possible. A large draw to this module is to automate mook behavior, but if they break targeting before the rolls are complete, then often damage won't be applied to the correct targets.

Many modules also handle target selection/deselection already, so by opening this up as an option, it would make mookAI much more compatible.

**Comment by QuantumFunks** [comment]
Yep agreed, this module is super awesome but unusable if it stops targeting before the attack
